parsing_2.py

```diff
@@ -85,13 +85,38 @@ def main():
             Redirect_count += 1
         elif line.status_code[0] == "4":
             Error_count += 1
-   
+
+
+    #Dictionary to store requests and how often they appear
+    requests_dict = {}
+
+    #basically copied from the example in general
+    #iterates through every request, if it exists it adds one, if not the request is added
+    #to the dictionary
+    for line in log_lines:
+        if line.request in requests_dict:
+            requests_dict[line.request] +=1
+        else:
+            requests_dict[line.request] = 1
+
+    #goes through the dictionary. If the number of requests is higher than max_requests,
+    #that number is stored in max_requests and the name in max_requests_name
+    max_requests = 0
+    max_requests_name = 'start'
+    for request in requests_dict:
+        if requests_dict[request] > max_requests:
+            max_requests = requests_dict[request]
+            max_requests_name = request
+
+
+    total_responses = len(log_lines)
```

Comment:
This appears to be unused

Reply:
I use line 112 it to calculate the percentage of redirect and error

Reply:
My bad ^^;